docs/src/pages/components/ScrollableCalendar/ScrollableCalendarPage.js

```diff
@@ -35,7 +36,7 @@ const ScrollableCalendarPage = () => (
       </IntroBlurb>,
     ]}
     webSubpage={<Web wrapped />}
-    nativeSubpage={null}
+    // webSubpage={<MarkdownPage content={Web} {...webMetadata} />}
```

Comment:
¿Que es el problemo con esta?

Reply:
Just saw the desciption - makes sense 👍

Reply:
Se queja de que `date-fns` no está disponible, por lo que debe agregarse como una dependencia en el paquete.

Reply:
NGL my Spanish is not advanced enough for that response 😂